app/src/main/java/pl/droidevs/books/appHelper/colors.java

```diff
@@ -0,0 +1,66 @@
+package pl.droidevs.books.appHelper;
+
+import android.graphics.Bitmap;
+import android.support.v4.graphics.ColorUtils;
+import android.support.v7.graphics.Palette;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Created by micha on 18.12.2017.
+ */
+
+public class colors {
+
+    public static final int HSL_HUE = 0;
+    public static final int HSL_SATURATION = 1;
+    public static final int HSL_LIGHT = 2;
+
+    private static final float MAX_ACTION_BAR_LIGHT = 0.7f;
+    private static final float MIN_ACTION_BAR_COLOR = 0.075f;
+
+    public static Palette.Swatch getDominantColor(Bitmap bitmap) {
+        List<Palette.Swatch> swatchesTemp = Palette.from(bitmap).generate().getSwatches();
+        List<Palette.Swatch> swatches = new ArrayList<>(swatchesTemp);
+        Collections.sort(swatches, (swatch1, swatch2) -> swatch2.getPopulation() - swatch1.getPopulation());
+
+        return swatches.size() > 0 ? swatches.get(0) : null;
+    }
+
+    public static int[] getColorsForBarsFromSwatch(Palette.Swatch swatch) {
+        if (swatch != null) {
```

Comment:
A tutaj to jest związane z tym, co @klisiewicz pisał o tablicy i obiekcie. Zamiast tworzyć tablicę z dwóch kolorów możesz mieć oddzielne metody zwracające każda odpowiedni kolor. np.: getStatusBarColor() i getActionBarColor()